Makefile

```diff
@@ -145,8 +145,7 @@ test_sim_app_profile:
 test_cover:
 	@export VERSION=$(VERSION); bash -x tests/test_cover.sh
 
-lint: tools ci-lint
-ci-lint:
```

Comment:
I think the assumption shouldn't be made that people have golangci-lint installed. I also think `make lint` is cleaner than `make tools lint`, but if this were to be merged then in places where we say make lint, we should put an extra line or disclaimer saying that they should run make tools lint the first time

Reply:
just saw chat in slack,`tools-all` is good